package.json

```diff
@@ -120,10 +103,398 @@
+        "./packages/victory-errorbar:build",
+        "./packages/victory-group:build",
+        "./packages/victory-histogram:build",
+        "./packages/victory-legend:build",
+        "./packages/victory-line:build",
+        "./packages/victory-pie:build",
+        "./packages/victory-polar-axis:build",
+        "./packages/victory-scatter:build",
+        "./packages/victory-selection-container:build",
+        "./packages/victory-shared-events:build",
+        "./packages/victory-stack:build",
+        "./packages/victory-tooltip:build",
+        "./packages/victory-voronoi:build",
+        "./packages/victory-voronoi-container:build",
+        "./packages/victory-zoom-container:build"
+      ]
+    },
+    "format": {
+      "dependencies": [
+        "format:root",
+        "format:pkgs"
+      ]
+    },
+    "format:root": {
+      "command": "nps format:root",
+      "files": [
+        ".eslintrc.js",
+        "*.js",
+        "*.json",
+        "scripts",
+        "config",
+        "demo",
+        "docs",
+        "stories",
+        "test"
+      ],
+      "output": []
+    },
+    "format:pkgs": {
+      "dependencies": [
+        "./packages/victory-native:format",
+        "./packages/victory-vendor:format",
+        "./packages/victory:format",
+        "./packages/victory-area:format",
+        "./packages/victory-axis:format",
+        "./packages/victory-bar:format",
+        "./packages/victory-box-plot:format",
+        "./packages/victory-brush-container:format",
+        "./packages/victory-brush-line:format",
+        "./packages/victory-candlestick:format",
+        "./packages/victory-canvas:format",
+        "./packages/victory-chart:format",
+        "./packages/victory-core:format",
+        "./packages/victory-create-container:format",
+        "./packages/victory-cursor-container:format",
+        "./packages/victory-errorbar:format",
+        "./packages/victory-group:format",
+        "./packages/victory-histogram:format",
+        "./packages/victory-legend:format",
+        "./packages/victory-line:format",
+        "./packages/victory-pie:format",
+        "./packages/victory-polar-axis:format",
+        "./packages/victory-scatter:format",
+        "./packages/victory-selection-container:format",
+        "./packages/victory-shared-events:format",
+        "./packages/victory-stack:format",
+        "./packages/victory-tooltip:format",
+        "./packages/victory-voronoi:format",
+        "./packages/victory-voronoi-container:format",
+        "./packages/victory-zoom-container:format"
+      ]
+    },
+    "lint": {
+      "dependencies": [
+        "lint:root",
+        "lint:pkgs"
+      ]
+    },
+    "lint:root": {
+      "command": "nps \"lint:base *.js .storybook scripts config demo stories test\"",
+      "files": [
+        ".eslintrc.js",
+        "*.js",
+        "scripts",
+        "config",
+        "demo",
+        "stories",
+        "typescript"
```

Comment:
Should this list should be in-sync with the `command` list? I think we need `.storybook` and `test` in here, and I don't think `typescript` is a valid file/folder.

Reply:
Um, ... YES!!!

For `test` I think VS Code autocompleted me to `typescript`. 😂 Fixing.